PowerEditor/src/WinControls/Preference/preference_rc.h

```diff
@@ -20,6 +20,7 @@
 	#define	PREF_MSG_ISCHECKED_GENERALPAGE (WM_USER + 1) // wParam:checkbox/radiobutton ID in General page. lParam is type of "bool *" to get result
 	#define	PREF_MSG_SETTOOLICONSFROMSTDTOSMALL (WM_USER + 2) 
 	#define	PREF_MSG_DISABLETABBARALTERNATEICONS (WM_USER + 3) 
+	#define PREF_MSG_SETTOOLICONSFROMSMALLTOSTD (WM_USER + 4) 

```

Comment:
Small remark: the previous entries have a tab between the #define keyword and the constant name, while yours has a space in-between

Reply:
@Mateos81, thanks for the review, I have changed it!